Cargo.lock

```diff
@@ -38,7 +38,7 @@ version = "0.2.14"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "d9b39be18770d11421cdb1b9947a45dd3f37e93092cbf377614828a319d5fee8"
 dependencies = [
- "hermit-abi",
+ "hermit-abi 0.1.19",
```

Comment:
Anyway we can avoid this duplication? If not, looks good to me.

Reply:
Interesting that cargo deny doesn't trip over this.  I assume because it's not used in the linux-x64 build.

In any case this is a lockfile-only update so it doesn't affect users of the library, only tests.